src/hooks-log-sandboxed.coffee

```diff
@@ -0,0 +1,12 @@
+hooksLog = require './hooks-log'
```

Comment:
It is needed, because for _sandboxed context libraries_ - only this file is required as `_log` property and used along the code.